docs/builds/guides/frameworks/react.md

```diff
@@ -216,6 +216,8 @@ const { styles } = require( '@ckeditor/ckeditor5-dev-utils' );
 
 Then, add two new elements to the exported object under the `module.rules` array (as new items of the `oneOf` array). These are SVG and CSS loaders required to handle the CKEditor 5 source:
 
+If you use the `postcss-loader` package in version 4+, use the snippet below:
```

Comment:
Does postcss-loader come with CRA? I can't see it mentioned anywhere in this guide.

Reply:
OK, I can see it comes with CRA. However, currently in version 3.